Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -263,6 +263,8 @@ public abstract class ApplicationHost : IServerApplicationHost, IDisposable
         /// <inheritdoc/>
         public bool ListenWithHttps => Certificate is not null && ConfigurationManager.GetNetworkConfiguration().EnableHttps;
 
+        public bool EnableMetrics => ConfigurationManager.Configuration.EnableMetrics || this._startupConfig.GetValue<bool>(EnableMetricsKey);
```

Comment:
Two things:

1. We don't use `this`
2. One should take priority over the other (probably the env var). With this expression, both must either not be set or be false to disable it.